terraform/main.tf

```diff
@@ -40,13 +40,13 @@ resource "random_string" "db_password" {
 }
 
 resource "aws_s3_bucket" "meadow_ingest" {
-  bucket = "${var.stack_name}-ingest"
+  bucket = "${var.stack_with_environment}-ingest"
```

Comment:
Can we do something here like `"${var.stack_name}-${var.environment}-ingest"` so we don't have to worry about whether the stack part of stack_with_environment matches?

Reply:
(Then `environment` would just be `s` or `p`)